requirements.txt

```diff
@@ -13,11 +13,8 @@ django-settings-export>=1.2.1,<1.2.99
 django-macros>=0.4.0,<0.4.99
 django-debug-toolbar>=2.0,<2.0.99
 
-# Force this version of promise because the newest version doesn't support dict
-promise==2.2.1
```

Comment:
Why isn't promise is updated to 2.3?

Reply:
its updated automatically to 2.3 as a dep to `graphene-django` 
In the older version of `graphene-django` I think it was set as >= 2.1 and in the latest version its set to >= 2.3

I can hard set it to 2.3 if you want

Reply:
If it is not needed then it's fine I was just wondering why you removed.